### PR TITLE
Download latest DE translations from Transifex

### PR DESCRIPTION
**Pull request by buren**
Doing this PR on behalf of @Komodorpudel.

### CODE REVIEW

**Comment by wbnns** [comment]
@buren Thanks. Could you please fix the failing test?

**Comment by Komodorpudel** [comment]
@wbnns and @buren Is the failing test caused by issues with the translation itself (wrong links etc.) ? If so, I would be glad to fix those issues.

File: _translations/de.yml [hunk 1]
```diff
@@ -575,33 +733,43 @@ de:
...
+    donationtxt: "Der einfachste Weg, um zu helfen, ist ein paar Bitcoins an BitGive zu <a href=\"http://bitgivefoundation.org/donate-now/\">spenden</a>. Sie können aber auch bei der Finanzierung von Projekten helfen, die mit Bitcoin im Zusammenhang stehen und von denen Sie glauben, dass sie in Zukunft nützlich sind."
     nonprofit: "Organisationen"
-    nonprofittxt: "Die <a href=\"https://bitcoinfoundation.org/\">Bitcoin Foundation</a> und viele andere <a href=\"#community##[community.non-profit]\">gemeinnützige Organisationen</a> sind dem Schutz und der Förderung von Bitcoin gewidmet. Sie können diesen Gruppen helfen in dem Sie Ihnen beitreten und an den Projekten, Diskussionen und Ereignissen teilhaben."
+    nonprofittxt: "Viele </a>gemeinnützige Organisationen</a> sind dem Schutz und der Förderung von Bitcoin gewidmet. Sie können diesen Gruppen helfen, indem Sie diesen beitreten und an Projekten, Diskussionen und Events mitwirken."
```
Comment (buren) [inline _translations/de.yml]: `</a>` should be removed there (in Transifex @Komodorpudel) then pushed here.

Comment (buren) [reply _translations/de.yml]: I managed to find the string ☝️ on Transifex here: https://www.transifex.com/bitcoinorg/bitcoinorg/viewstrings/#de/bitcoinorg/20474254?key=support-bitcoin.nonprofittx

Comment (Komodorpudel) [reply _translations/de.yml]: Hey Jacob,

Thank you very much!
I went through all the warnings and corrected every faulty link I could found.

I hope that there are no mistakes left now 😊

**Comment by buren** [comment]
@Komodorpudel seems like you squashed some of the errors but actually caused two additional ones :/

```
- ./_site/de/community.html
  *  275:83: ERROR: htmlParseEntityRef: no name
- ./_site/de/entwicklung.html
  *  297:88: ERROR: htmlParseEntityRef: no name
- ./_site/de/presse.html
  *  267:108: ERROR: Unexpected end tag : a
  *  267:79: ERROR: Unexpected end tag : a
```

The error in `entwicklung.html` and `presse.html` are new 🤔

**Comment by Komodorpudel** [comment]
@burem 

I’m really sorry! I just went through all the string and holy fuck:

I did not create new errors. They were already there just like quite a few others.

At some point, links in the original English version were probably replaced and pushed to Transifex.
It seems like a lot of german translations for those strings got “unreviewed” in the process. Consequentially, I corrected the links for most of them.
However, a few stayed “reviewed”, which is probably also the reason for the missing warning messages. 

Anyhow, I skimmed all string, correcting every issue I could find. Given the large amount of text, I will check everything again later today or tomorrow. I will let you know as soon as I’m done.

**Comment by Komodorpudel** [comment]
@buren Sorry for the delay!

I checked everything again and could not find any other mistakes!
However, I changed a few minor things in some strings. But I assume that this does not create new errors.

Thank you!

**Comment by Komodorpudel** [comment]
@buren  How does it look? Still mistakes left?

**Comment by wbnns** [comment]
@buren Thanks for trying to work on this and apologies for the delay (@Komodorpudel Thanks also for the help). We'll grab this out of Transifex and will update it.

**Comment by buren** [comment]
Awesome @wbnns let me know if there is anything else I can do.

**Comment by Komodorpudel** [comment]
Heyho, I guess this chapter is now finally closed :) Thanks guys!

**Pull request closed without merging**